Class/Query.php

```diff
@@ -222,24 +223,23 @@ public function editTest($id)
      * 
      * @return void
      */
-    public function updateTest($submit, $Question, $Answer, $id)
+    public function updateTest($Question, $Answer, $id)
```

Comment:
@Ratheesh392 Follow the variable naming convention. Always start with small letter or underscores